phpstan.neon

```diff
@@ -16,7 +16,7 @@ parameters:
         - config
     inferPrivatePropertyTypeFromConstructor: true
     bootstrapFiles:
-        - bin/.phpunit/phpunit-8.3-0/vendor/autoload.php
+        - vendor/bin/.phpunit/phpunit-8.3-0/vendor/autoload.php
```

Comment:
On my local machine, there is no `vendor/bin/.phpunit/` directory, just `bin/.phpunit/` ... I can't see anything in this pull request that would change that behaviour?

Reply:
the phpunit composer script was changed from `"phpunit": "bin/phpunit"` to `"phpunit": "SYMFONY_PHPUNIT_VERSION=8.3 vendor/bin/simple-phpunit"`. maybe that is the reason?

Reply:
could be, yes. Why not using `bin/phpunit`? Is there a disadvantage using that?

Reply:
we are installing `symfony/phpunit-bridge` instead of `phpunit/phpunit`. so i think we should also use the wrapper provided by `symfony/phpunit-bridge` 🙂  https://symfony.com/doc/current/components/phpunit_bridge.html

Reply:
Ah, I understand 🙂

Reply:
dont know the exact differences to be honest. but as far as i understood, using the `symfony/phpunit-bridge` is the symfony-way for using phpunit